frameworks/PHP/ubiquity/benchmark_config.json

```diff
@@ -243,6 +243,25 @@
         "display_name": "ubiquity-ngx-raw",
         "notes": "",
         "versus": "ngx_php"
+      },
+      "ngx-micro": {
+        "json_url": "/micro/json",
+        "plaintext_url": "/micro/plaintext",
+        "port": 8080,
+        "approach": "Realistic",
+        "classification": "Fullstack",
```

Comment:
classification: micro !!!

Reply:
Yes, I also thought about it.
But the framework remains the same, and its configuration has not changed, unlike those who offer a truly different micro version.